ansible_ai_connect/ai/api/wca/model_id_views.py

```diff
@@ -234,6 +237,11 @@ def do_validated_operation(request, api_key_provider, model_id_provider, on_succ
 
         return on_success(organization.id, model_id)
 
+    except WcaEmptyResponse:
+        # An empty response does not mean the model_id is invalid.
```

Comment:
can we log that we have got an empty response here?

Reply:
We log all exceptions [here](https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/ai/api/model_client/wca_utils.py#L48-L57).

Reply:
but here you are catching the empty response exception and return `on_success` -- you are not raising it again, how will the exception be handled again?

Reply:
For the sole purpose of validating a `model_id` a `WcaEmptyResponse` exception is considered a valid response and hence the exception is swallowed and the `on_success` invoked. Elsewhere, for example [`explanations`](https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/ai/api/views.py#L704-L709), the exception is caught and raised again as an HTTP error.

Reply:
@jameswnl Logging added.